pkg/agent/hooks/structs/structs.go

```diff
@@ -5,15 +5,6 @@ package structs
 
 type BpfSpinLock struct{ Val uint32 }
 
-// struct dest_info_t
```

Comment:
These were the references of C structure in ebpf code, Please keep the commented code.